homeassistant/components/isy994/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "Isy994",
   "documentation": "https://www.home-assistant.io/components/isy994",
   "requirements": [
-    "PyISY==1.1.1"
+    "https://github.com/shbatm/PyISY/archive/develop.zip#PyISY==1.1.2"
```

Comment:
This needs to be fixed before merge.

Reply:
Agreed. See comment in original PR description.  I am not the owner of PyISY and have been trying to give him the time to review and accept the PR over there. This cannot be accepted until that change is integrated.  I will give it 1 more week before I push a forked version to PyPi.

Reply:
I updated the lines for now and split off my testing version to another branch.

##PLEASE DO NOT MERGE UNITL PyISY HAS BEEN UDPATED